Add details for TCP log collection (#11454)
DataDog/documentation: content/en/agent/logs/_index.md

{{% tab "TCP/UDP" %}}

> To gather logs from your `<APP_NAME>` application that forwards its logs with to TCP port **10518**, create a `<APP_NAME>.d/conf.yaml` file at the root of your [Agent's configuration directory][1] with the following content:

Reviewer: ```suggestion
To gather logs from your `<APP_NAME>` application that forwards its logs to TCP port **10518**, create a `<APP_NAME>.d/conf.yaml` file at the root of your [Agent's configuration directory][1] with the following content:
```

If you are using Serilog, `Serilog.Sinks.Network` is an option for connecting with UDP.

> As of Agent version 7.31.0, the Agent keeps a TCP connection open indefinitely.

Reviewer: ```suggestion
In the Agent version 7.31.0+, the TCP connection stays open indefinitely even when idle.
```

> As of Agent version 7.31.0, the agent will keep a TCP connection open indefinitely.

Reviewer: ```suggestion
As of Agent version 7.31.0, the Agent keeps a TCP connection open indefinitely.
```

> Previous versions closed conections after 60 seconds without log messages.

Reviewer: This was not documented before; let's not document it now.

```suggestion
```

Author: The sentence before is pretty vague without this counterpoint.  Maybe I could add " open indefinitely, even if idle."?

Or, maybe this doesn't need to be documented at all, if what we think of as "removing a feature" is actually "removing a bug" from the user's perspective. WDYT?

Reviewer: I think if we rephrased the first sentence as "In the Agent version 7.31.0+, the TCP connection stays open indefinitely even when idle." we can delete the rest of this!

> Previous versions closed conections after 60 seconds without log messages.
> On connection failure, your application should reconnect to the Agent.

Reviewer: ```suggestion
```

> On connection failure, your application should re-connect to the agent.

Reviewer: ```suggestion
On connection failure, your application should reconnect to the Agent.
```
